packages/playwright-core/src/utils/isomorphic/locatorGenerators.ts

```diff
@@ -50,15 +50,6 @@ export function asLocators(lang: Language, selector: string, isFrameLocator: boo
 
 function innerAsLocators(factory: LocatorFactory, parsed: ParsedSelector, isFrameLocator: boolean = false, maxOutputSize = 20): string[] {
   const parts = [...parsed.parts];
-  // frameLocator('iframe').first is actually "iframe >> nth=0 >> internal:control=enter-frame"
```

Comment:
It's suspicious that you can remove this without breaking anything. Let's leave it and the other place below? you can remove it in a separate change